gulpfile.js/config.js

```diff
@@ -1,12 +1,13 @@
 const srcDir = './src'
+const publicDir = './public'
 
 module.exports = {
   'src': srcDir,
   'dest': './static',
-  'public': './public',
+  'public': publicDir,
   'env': {
     'prod': {
-      'baseUrl': 'https://docs.flyntwp.com/'
+      'baseUrl': 'http://localhost:1313/'
```

Comment:
This whole env stuff and the setBase task can probably be removed now that we handle setting the base url in the jenkins build..